pymongoexplain/__main__.py

```diff
@@ -48,6 +49,18 @@ def new_func(self: Collection, *args, **kwargs):
     setattr(Collection, old_func_name, make_func(old_func, old_func_name))
 
 if __name__ == '__main__':
-    for file in sys.argv[1:]:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "input_script", action="store", nargs=1, help="The script that you "
```

Comment:
I don't think there is any point in having `action=store` here. This is the default behavior.

`nargs=1` is also not needed as that is what it would default to if you used the default action.

Reply:
leaving off "nargs=1" results in each character being parsed into a list. It is necessary for it to be parsed into a string.